python/lambdas/copy-files-from-tdr/lambda_function_test.py

```diff
@@ -8,43 +8,16 @@
 @patch.dict('os.environ', {'DESTINATION_BUCKET': 'destination-bucket'})
 @patch.dict('os.environ', {'DESTINATION_QUEUE': 'destination-queue'})
 class TestLambdaFunction(unittest.TestCase):
-    @patch('lambda_function.s3_client.head_object')
-    @patch('lambda_function.s3_client.copy_object')
-    @patch('lambda_function.sqs_client.send_message')
-    def test_standard_copy(self, mock_send_message, mock_copy_object, mock_head_object):
-        mock_head_object.return_value = {'ContentLength': (5 * 1024 * 1024 * 1024) - 1}
-        event = {
-            'Records': [
-                {'body': '{"bucket": "source-bucket","fileId":"test-file"}'}
-            ]
-        }
-        context = {}
-        lambda_function.lambda_handler(event, context)
-
-        def assert_copy_object_arguments(idx, key):
-            expected_request = \
-                {'Bucket': 'destination-bucket', 'CopySource': {'Bucket': 'source-bucket', 'Key': key}, 'Key': key}
-            self.assertEqual(mock_copy_object.call_args_list[idx][1], expected_request)
-
-        expected_sqs_args = \
-            {'MessageBody': '{"location": "s3://destination-bucket/test-file"}', 'QueueUrl': 'destination-queue'}
-
-        self.assertEqual(mock_copy_object.call_count, 2)
-        assert_copy_object_arguments(0, 'test-file')
-        assert_copy_object_arguments(1, 'test-file.metadata')
-        self.assertEqual(mock_send_message.call_args_list[0][1], expected_sqs_args)
-
     @patch('lambda_function.s3_client.head_object')
     @patch('lambda_function.s3_client.create_multipart_upload')
```

Comment:
Is the reason why you have not removed these is because boto3 calls these?

Reply:
I think it does yeah. I did remove them and it stopped working so I think so.